script/vsts/platforms/templates/bootstrap.yml

```diff
@@ -0,0 +1,31 @@
+steps:
+  - pwsh: |
+      # OS specific env variables
+      if ($env:AGENT_OS -eq "Windows_NT") {
+        $env:NPM_BIN_PATH="C:/hostedtoolcache/windows/node/12.13.1/x64/npm.cmd"
+        $env:npm_config_build_from_source=true
+      }
+      if ($env:AGENT_OS -eq "Darwin") {
+        $env:NPM_BIN_PATH="/usr/local/bin/npm"
+        $env:npm_config_build_from_source=true
+      }
+      if ($env:AGENT_OS -eq "Linux") {
+        $env:NPM_BIN_PATH="/usr/local/bin/npm"
+        $env:CC=clang
+        $env:CXX=clang++
+        $env:npm_config_clang=1
+      }
+
+      # Bootstrap
+      if (($env:AGENT_OS  -eq "Windows_NT") -and ($env:BUILD_ARCH -eq "x86")) {
+        node 'script\vsts\windows-run.js' 'script\bootstrap.cmd'
+      }
+      else {
+        script/bootstrap
+      }
+    displayName: Bootstrap build environment
+    env:
+      GITHUB_TOKEN: $(GITHUB_TOKEN)
+      CI: true
+      CI_PROVIDER: VSTS
+      condition: or(ne(variables['MainNodeModulesRestored'], 'true'), ne(variables['ScriptNodeModulesRestored'], 'true'), ne(variables['ApmNodeModulesRestored'], 'true'))
```

Comment:
This indentation makes `condition` and env variable. Bootstrapping always runs.

We should indent it two spaces less, toward the left.

Reply:
Thanks for catching this. I will fix it!